teachopencadd/talktorials/002_compound_adme/talktorial.ipynb

```diff
@@ -6,11 +6,11 @@
    "source": [
```

Comment:
<p>Add one more sentence for CADD context:  </p><p>In the context of drug design, it is important to filter candidate molecules by e.g. their physicochemical properties. In this talktorial, the compounds acquired from ChEMBL (<strong>Talktorial 001</strong>) will be filtered by Lipinsik's rule of five to keep only orally bioavailable compounds (applying Lipinski's rule of five).</p><p>-&gt; change </p><p>-- 'active' to 'bioavailable'</p><p>-- 'check for' to 'keep only'</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/volkamerlab/TeachOpenCADD/pull/49/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='2'/>

Reply:
<p>Done.</p>